openapi.yaml

```diff
@@ -625,65 +645,34 @@ components:
       type: object
       x-examples:
         example-1:
+          account_key: "3fa85f64-5717-4562-b3fc-2c963f66afa6"
```

Comment:
User-service will not store the accountKey. AccountKeys are important only during authentication/auhtorization. Other services must (!) have no notion of this or even get access to them.